contrib/buildinfo/test/src/mill/contrib/buildinfo/BuildInfoTests.scala

```diff
@@ -17,6 +18,8 @@ object BuildInfoTests extends TestSuite {
     def scalaVersion = scalaVersionString
     def buildInfoPackageName = "foo"
     def buildInfoMembers = Seq.empty[BuildInfo.Value]
+
+    def millDiscover = Discover[this.type]
```

Comment:
Shouldn't this be a `val` or `lazy val`. The object is already at the end of the inheritance hierarchy, and the `def` may cause wasteful re-initialisation of `Discover`.

Reply:
Probably can be a lazy val